app/controllers/claims/schools/claims_controller.rb

```diff
@@ -33,6 +33,14 @@ def update
     end
   end
 
+  def confirm; end
+
+  def submit
+    Claims::Submit.call(claim: @claim)
```

Comment:
This should reference the model, `Claim`, rather than the service namespace, `Claims`.

This is because we're attempting to `Submit` the `Claim`.

```suggestion
    Claim::Submit.call(claim: @claim)
```